### ISSUE TITLE
Reports being written with null or empty fields

### ISSUE THREAD

**Issue by GoogleCodeExporter** [useataaaaa/acra#62]
```
What steps will reproduce the problem?
1. Causing ACRA to be called when the PackageManager ha died for your App, such 
as when it is being reinstalled.

What is the expected output? What do you see instead?
ACRA should log all fields that don't directly rely on the PackageManager or 
services mediated by the app Context.
Instead ACRA throws an Exception when it encounters a dead PackageManager 
causing most crash fields to not be filled in.

```

Original issue reported on code.google.com by `william....@gmail.com` on 16 Jul 2011 at 7:17

**Comment by GoogleCodeExporter** [comment]
```
Fixed in Rev#720
```

Original comment by `william....@gmail.com` on 16 Jul 2011 at 7:23
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `kevin.gaudin` on 7 Sep 2012 at 9:37
- Changed state: **Verified**
